pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go

```diff
@@ -1488,9 +1488,9 @@ func TestMultipleConstraints(t *testing.T) {
 		},
 		{
 			// 1. to fulfil "zone" constraint, incoming pod can be placed on zone2 (node-x or node-y)
-			// 2. to fulfil "node" constraint, incoming pod can be placed on node-b or node-x
+			// 2. to fulfil "node" constraint, incoming pod can be placed on node-a, node-b or node-x
 			// intersection of (1) and (2) returns node-x
-			name: "Constraints hold different labelSelectors, spreads = [1/0, 1/0/0/1]",
+			name: "Constraints hold different labelSelectors, spreads = [1/0, 0/0/0/1]",
```

Comment:
Pod `p-a1` is located on node-a.

Reply:
Due to the constraints is defined as this,
```
SpreadConstraint(1, "node", v1.DoNotSchedule, st.MakeLabelSelector().Exists("bar").Obj()).
```
and this pod is defined as this,
```
st.MakePod().Name("p-a1").Node("node-a").Label("foo", "").Obj(),
```
constraint selector doesn't match with the pod label,

Double confirmed by,

```
(dlv) p s.TpPairToMatchNum
map[k8s.io/kubernetes/pkg/scheduler/framework/plugins/podtopologyspread.topologyPair]*int32 [
        {key: "zone", value: "zone2"}: *0,
        {key: "node", value: "node-y"}: *1,
        {key: "zone", value: "zone1"}: *1,
        {key: "node", value: "node-b"}: *0,
        {key: "node", value: "node-x"}: *0,
        {key: "node", value: "node-a"}: *0,
]
```

Reply:
I know, but that comment just literally describes the fact how the pods are located across the cluster - it's not related with incoming pod's constraints.

Reply:
okay, I will revert this line back.

Reply:
Done